fsevents.go

```diff
@@ -149,47 +94,37 @@ func DeviceForPath(path string) (int32, error) {
 	return stat.Dev, nil
 }
 
-// EventIDForDeviceBeforeTime returns an event ID before a given time.
-func EventIDForDeviceBeforeTime(dev int32, before time.Time) uint64 {
-	tm := C.CFAbsoluteTime(before.Unix())
-	return uint64(C.FSEventsGetLastEventIdForDeviceBeforeTime(C.dev_t(dev), tm))
-}
-
-/*
-
-	Primary EventStream interface.
-	You can provide your own event channel if you wish (or one will be created
-	on Start).
-
-	es := &EventStream{Paths: []string{"/tmp"}, Flags: 0}
-	es.Start()
-	es.Stop()
-
-*/
-
-// EventStream is the primary interface to FSEvents.
+// EventStream is the primary interface to FSEvents
+// You can provide your own event channel if you wish (or one will be
+// created on Start).
+//
+// es := &EventStream{Paths: []string{"/tmp"}, Flags: 0}
```

Comment:
This code snippet doesn't render so well in godoc. not sure if it could be made into a (working) [example](https://blog.golang.org/examples)? though an example need not be executed, so long as it compiles

http://127.0.0.1:8080/pkg/github.com/fsnotify/fsevents/

![godoc](https://cloud.githubusercontent.com/assets/4566/19335718/9d92216e-90c2-11e6-9307-9a5b4672cc72.png)